astronomer/providers/amazon/aws/example_dags/example_aws_nuke.py

```diff
@@ -46,6 +46,13 @@
         f"aws-nuke -c /usr/local/airflow/dags/nuke-config.yml --profile default --force --no-dry-run; ",
     )
 
+    delete_stale_emr_vpcs = BashOperator(
+        task_id="delete_stale_emr_vpcs",
+        bash_command="sh $AIRFLOW_HOME/dags/delete_stale_emr_vpcs.sh ",
+        trigger_rule="all_done",
+    )
+
     end = DummyOperator(task_id="end")
 
-    start >> terminate_running_emr_virtual_clusters >> execute_aws_nuke >> end
+    start >> terminate_running_emr_virtual_clusters >> execute_aws_nuke >> delete_stale_emr_vpcs
```

Comment:
May I know why we can't run `delete_stale_emr_vpcs` and `execute_aws_nuke` in parallel? Thanks

Reply:
Ideally, aws_nuke should take care of deleting the VPCs too. But, since we observed that sometimes they remain stale, I am running them after the aws nuke. I do not want to delete the VPCs before running AWS nuke because it may affect the nuke if VPCs are deleted before and then the CloudFormation tries deleting them as part of its stack which may fail.